geometry/jarvis_algorithm.cpp

```diff
@@ -0,0 +1,177 @@
+/**
+ * @file jarvis_algorithm.cpp
```

Comment:
Better to remove the file name.
```suggestion
 * @file
```